languages/cpp/exercises/concept/basics/.docs/introduction.md

```diff
@@ -0,0 +1,43 @@
+C++ is a statically-typed language, which means that everything has a type at compile-time. Assigning a value to a name is referred to as defining a variable. A variable can be defined either by explicitly specifying its type, or by letting the C++ compiler infer its type based on the assigned value (known as _type inference_). Therefore, the following two variable definitions are equivalent:
+
+```cpp
+int explicitVar = 10; // Explicitly typed
+auto implicitVar = 10; // Implicitly typed
+```
+
+Updating a variable's value is done through the `=` operator. Once defined, a variable's type can never change.
```

Comment:
Probably worth explicitly naming the operator. This sets things up nicely for later on when the similar looking equality operator is introduced, i.e. using the proper names makes the difference more apparent.

```suggestion
Updating a variable's value is done through the `=` assignment operator. Once defined, a variable's type can never change.
```

Reply:
That makes sense. :+1: